oracle/oracles/pd.go

```diff
@@ -52,13 +53,16 @@ import (
 var _ oracle.Oracle = &pdOracle{}
 
 const slowDist = 30 * time.Millisecond
+const minTimestampUpdateInterval = 50 * time.Millisecond
+const maxTimestampUpdateInterval = 5 * time.Second
 
 // pdOracle is an Oracle that uses a placement driver client as source.
 type pdOracle struct {
 	c pd.Client
 	// txn_scope (string) -> lastTSPointer (*atomic.Pointer[lastTSO])
-	lastTSMap sync.Map
-	quit      chan struct{}
+	lastTSMap      sync.Map
+	quit           chan struct{}
+	updateInterval time.Duration
```

Comment:
Not related to this PR, I think we can name `updateInterval` more clearly, like `lastTSUpdateInterval`, because `updateInterval` is a little vague about what to be updated.

Reply:
I can update for clearer naming